lib/instances/ContextActionService.lua

```diff
@@ -1,4 +1,15 @@
-local BaseInstance = import("./BaseInstance")
-local ContextActionService = BaseInstance:extend("ContextActionService")
-
+local BaseInstance = import("./BaseInstance")
+local validateType = import("../validateType")
+local ContextActionService = BaseInstance:extend("ContextActionService")
+
+function ContextActionService.prototype:BindCoreAction(actionName, functionToBind, createTouchButton, ...)
+	validateType("actionName", actionName, "string")
+	validateType("functionToBind", functionToBind, "function")
+	validateType("createTouchButton", createTouchButton, "boolean")
+end
+
+function ContextActionService.prototype:UnbindCoreAction(actionName)
+	validateType("actionName", actionName, "string")
+end
```

Comment:
Can we leave notes on these functions regarding further validation? For example, is it an error to unbind an action that isn't bound, or bind an action that's already bound?

Reply:
Apparently none of those things are errors. 

It looks like unbinding non-existent actions is fine, and binding over an existing action name just overwrites the same binding. Binding new actions to the same inputs does the stacking thing that CAS is supposed to. I was testing this with `BindAction` and `UnbindAction` rather than their `Core` analogues though, so I could be wrong.

I'll leave some comments either way.

Reply:
You (maybe wouldn't) be surprised how little type checking on parameters is intentional at Roblox.